synthea-hiv/uploader/fhir_client.py

```diff
@@ -45,18 +45,18 @@ class OpenMrsClient:
 
   def __init__(self, base_url: str):
     self.base_url = base_url
-    self.session = _setup_session(self.base_url)
-    self.session.auth = ('admin', 'Admin123')
+    self.auth=('admin', 'Admin123')
+    self.headers = {'Content-Type': 'application/json'}
     self.response = None
 
   def post_single_resource(self, resource: str, data: Dict[str, str]):
     url = f'{self.base_url}/{resource}'
-    response_ = self.session.post(url, json.dumps(data))
+    response_ = requests.post(url, headers=self.headers, data=json.dumps(data), auth=self.auth)
     self.response = _process_response(response_)
 
   def get_resource(self, resource: str):
     url = f'{self.base_url}/{resource}'
-    response_ = self.session.get(url)
```

Comment:
There seem to be other references to `session` in the `HapiClient` subclass, should those be updated too?